src/feedback/commands/submitFeedback.ts

```diff
@@ -26,16 +26,18 @@ export function submitFeedback(listener?: (message: any) => Promise<void>): vsco
     const baseTemplateFn = _.template(BaseTemplates.SIMPLE_HTML)
 
     panel.webview.html = baseTemplateFn({
+        cspSource: panel.webview.cspSource,
         content: '<h1>Loading...</h1>',
     })
 
     const feedbackTemplateFn = _.template(FeedbackTemplates.SUBMIT_TEMPLATE)
 
-    const loadScripts = ExtensionUtilities.getScriptsForHtml(['submitFeedbackVue.js'])
-    const loadLibs = ExtensionUtilities.getLibrariesForHtml(['vue.min.js'])
-    const loadStylesheets = ExtensionUtilities.getCssForHtml(['submitFeedback.css'])
+    const loadScripts = ExtensionUtilities.getScriptsForHtml(['submitFeedbackVue.js'], panel.webview)
+    const loadLibs = ExtensionUtilities.getLibrariesForHtml(['vue.min.js'], panel.webview)
+    const loadStylesheets = ExtensionUtilities.getCssForHtml(['submitFeedback.css'], panel.webview)
```

Comment:
Some styles or something may be off in the template (sample image attached)

![image](https://user-images.githubusercontent.com/39839589/83181917-67064400-a0da-11ea-8091-bc24924e1d99.png)

Reply:
Did a dumb thing and only migrated the template for Lambda (which I tested on) and forgot the rest. Fix incoming...